packages/lib/src/utils/FileSystem.ts

```diff
@@ -42,7 +42,11 @@ export function ifNotExistsThrow(filename: string, message: string): void {
 
 // TS-TODO: Returned object could be of a more specific type
 export function parseJson(filename: string): any {
-  return JSON.parse(read(filename));
+  try {
+    return JSON.parse(read(filename));
+  } catch(error) {
+    return null;
+  }
```

Comment:
I prefer to fail loudly if we are trying to parse a file that is not a valid JSON, or does not exist. It also helps to more descriptive error messages (such as "file not exists" or "invalid JSON at line N", instead of "cannot read property X of null"). I understand that this helps on the `getContractNames` method, but I'd rather check for json files there.